pages/signup.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from "react";
+import { AnimatePresence } from "framer-motion";
+
+import type { NextPage } from "next";
+import { Container, HomeLayout } from "@/components";
+import Step1 from "@/components/signup/Step1";
+import Step2 from "@/components/signup/Step2";
+import Step3 from "@/components/signup/Step3";
```

Comment:
have an `index.ts` in `@/components/signup` and have step1, 2, and 3 exported from there.

e.g
```ts
export { default as Step1 } from "./Step1";
...
```